assets/src/components/lateView.tsx

```diff
@@ -503,7 +506,7 @@ const LateBusRow = ({
         />
       </td>
       <td className="c-late-view__adherence-cell">
-        {secondsToMinutes(vehicle.scheduleAdherenceSecs) * -1}
+        {secondsToMinutes(vehicle.scheduleAdherenceSecs || 0) * -1}
```

Comment:
Good question - we already filter out buses with `scheduleAdherenceSecs` of `null` (see line 279), so in effect we're able to make an assumption about the data that the type system can't. That said I think this is another point in favor of splitting invalid vehicles out into their own type as discussed elsewhere, so that we could explicitly work with not-invalid buses and avoid this sort of logic.